plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -29,6 +29,7 @@ global proc mayaUSDRegisterStrings() {
     register("kCreateUsdStageFromFile", "Create USD Stage from File");
     register("kCreateUsdStageFromFileOptions", "Create USD Stage from File Options");
     register("kCreateStageFromFile", "Create Stage from File");
+    register("kCreate", "Ok");
```

Comment:
This **kCreate** ID is already being registered by the Layer Editor here: https://github.com/Autodesk/maya-usd/blob/dev/lib/usd/ui/layerEditor/stringResources.h#L58

I think the problem you are having is because you aren't building with Qt, and thus no layer editor. Which is a supported build option.

However this ID is being used by the plugin here: https://github.com/Autodesk/maya-usd/blob/dev/plugin/adsk/scripts/mayaUsd_createStageFromFile.mel#L224

That is the error you are seeing. However note the string should be "Create" not "Ok".  So what I would suggest is to change this to:
```suggestion
    register("kCreateStage", "Create");
```
And then change line 224 in `mayaUsd_createStageFromFile.mel` to use kCreateStage.

Reply:
Yes, thank you.
Changed.